### DISCUSSION TITLE
Reducing scope/splitting the repo

### DISCUSSION THREAD

**Discussion by Sephsekla** [WordPress/wp-feature-notifications#551] [General]
It's no secret that the project has stalled in the last year or so, due to a lack of contributors but also a pretty wide scope.

In order to get the project moving again, as well as to reduce potential roadblocks with conflicts with other admin UI projects, I'd like to propose the following scope changes:

- JS/UX work is split out into a separate repo. This should make use of the data model and REST APIs
- This repo should focus on:
    - Database model
    - PHP API
    - REST API
- Another repo can focus on:
    - JS Dashboard components (as already seen)
    - Storybook reference

I think this is a far more manageable, focused scope, that should be far more achievable.

**Comment by TheRadicalDreamer** [comment]
This should be open again :(

Now Woocommerce has a notification system, maybe WordPress can use the same.
